scripts/start-services-integration.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
```

Comment:
This is the new "we can run the script from this repo but also edit and see the changes without committing and merging through the ui repo".
It is exceedingly simple by design as it just drops into the ui and api repositories and spins them up with all their dependencies using their own docker files. This splits the responsibility between the e2e tests and the solutions. This test repo now gets to say "I need to boot you up" with a chance to overwrite any setting if need be whilst the solutions themselves are responsible for "how they boot up, you can declare settings for me".